x-pack/plugin/ent-search/qa/rest/build.gradle

```diff
@@ -7,7 +7,7 @@ dependencies {
 
 restResources {
   restApi {
-    include '_common', 'cluster', 'nodes', 'indices', 'index', 'search_application'
+    include '_common', 'cluster', 'nodes', 'indices', 'index', 'search_application', 'xpack.usage'
```

Comment:
Is this necessary? It doesn't seem to be included in other module gradle builds.

Reply:
if i remove it this is what i get:

> java.lang.RuntimeException: Failure at [entsearch/100_usage:30]: Rest api [xpack.usage] cannot be found in the rest spec. Either it doesn't exist or is missing from the test classpath. Check the 'restResources' block of your project's build.gradle file.

Reply:
Thanks for verifying!